.happy/config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "config_version": "v1",
```

Comment:
Just curious, what requires a version bump in the happy config?

Reply:
This was done to run rdev so I could test the api in the remote dev environment! I am changing it back in my next commit because it was causing issues with running the local environment